Common/SonarQube.Plugins.Common.csproj

```diff
@@ -30,8 +30,56 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
*Plugins.Common* doesn't seem like the right place to put Roslyn-specific code. It would have been better to merge *Roslyn.RuleGenerator* and *Roslyn.PluginGenerator* into a single assembly.